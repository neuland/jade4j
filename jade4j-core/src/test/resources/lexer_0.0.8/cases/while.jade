- var x = 1;
ul
  while x < 10
    - x++;
    li= x
