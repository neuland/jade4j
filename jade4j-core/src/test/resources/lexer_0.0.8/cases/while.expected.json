{"type":"code","line":1,"val":"var x = 1;","escape":false,"buffer":false}
{"type":"newline","line":2}
{"type":"tag","line":2,"val":"ul","selfClosing":false}
{"type":"indent","line":3,"val":2}
{"type":"code","line":3,"val":"while (x < 10)","requiresBlock":true}
{"type":"indent","line":4,"val":4}
{"type":"code","line":4,"val":"x++;","escape":false,"buffer":false}
{"type":"newline","line":5}
{"type":"tag","line":5,"val":"li","selfClosing":false}
{"type":"code","line":5,"val":"x","escape":true,"buffer":true}
{"type":"outdent","line":6}
{"type":"outdent","line":6}
{"type":"eos","line":6}